cuqipy_fenics/pde.py

```diff
@@ -18,10 +19,23 @@ class FEniCSPDE(PDE,ABC):
 
     Parameters
     ----------
-    PDE_form : python function handle
-        Function handle of a python function that returns the FEniCS weak form of the PDE.
-
-        This python function takes as input, in this given order, the Bayesian parameter (input of the forward model), the state variable (solution variable), the adjoint variable (which is also the test variable in the weak formulation) as FEniCS functions (or FEniCS trail or test functions). See, for example, `demos/demo03_poisson_circular.py` for an example of how to define this function.
+    PDE_form : python function handle or a tuple of two python function handles
+        Function handle of a python function that returns the FEniCS weak form
+        of the PDE. Alternatively, a tuple of function handles can be provided
+        to specify the PDE weak form left hand side as the first element of the 
+        tuple, and the weak form right hand side as the second element of the
+        tuple.
+
+        The python function representing the full form or the left hand side
+        takes as input, in this given order, the Bayesian parameter (input of
+        the forward model), the state variable (solution variable), the adjoint
+        variable (which is also the test variable in the weak formulation) as
+        FEniCS functions (or FEniCS trail or test functions). See, for example,
+        `demos/demo03_poisson_circular.py` for an example of how to define the
+        full form.
```

Comment:
This is a rather confusing text. Any chance you can simply it? Perhaps with small examples or something? Sometimes its easier if you simply write the options down as examples.

Reply:
Thank you for pointing this out, I rewrote this part and added a code example.